operator/go.mod

```diff
@@ -246,6 +246,7 @@ require (
 	github.com/prometheus/client_model v0.6.2 // indirect
 	github.com/prometheus/procfs v0.16.1 // indirect
 	github.com/quasilyte/go-ruleguard/dsl v0.3.22 // indirect
+	github.com/redpanda-data/redpanda-operator/charts/console/v3 v3.1.0 // indirect
```

Comment:
Why this indirect import is present if we have 
```
github.com/redpanda-data/redpanda-operator/charts/console v0.0.0-20250718150737-e01f8476d560
```

Reply:
The v3 dep is indirect through the redpanda v25 chart.

So:
```
operator --> Console v2 (JSON compat tests)
           | ---> Redpanda v25 --> Console v3 (Through rendering)
```

Good catch though, I didn't realize it until you pointed it out!